src/backends/tensorflow/base_backend.cc

```diff
@@ -410,6 +410,16 @@ ReadStringOutputTensor(
 {
   size_t tensor_element_idx = 0;
 
+#ifdef TRTIS_ENABLE_GPU
+  bool cuda_copy = false;
+  cudaStream_t stream;
+  cudaError_t cuerr = cudaStreamCreate(&stream);
```

Comment:
This is really bad to be creating a stream every time we copy a string output tensor. Don't we already have a stream created for each backend for other copy tasks? This should be dealt with in the copy cleanup.